source/introduction/history-of-ethereum.rst

```diff
@@ -1,3 +1,70 @@
+
+DEVcon-0
+--------------------------------------------------------------------------------
+In November 2014, ETH DEV organized the `DEVcon-0 event <https://blog.ethereum.org/2014/12/05/d%CE%BEvcon-0-recap/>`_, which brought together Ethereum developers from around the world to Berlin to meet and discuss a diverse range of Ethereum technology topics. Several of the presentations and sessions at DEVcon-0 would later drive important initiatives to make Ethereum more reliable, more secure, and more scalable. Overall, the event galvanized developers as they continued to work towards the launch of Ethereum. 
+
+ETH DEV Bounty Hunters
+--------------------------------------------------------------------------------
+In early 2015, a bug bounty program was launched through `ETH DEV Bounty Hunters website <http://bounty.ethdev.com/>`_ offering BTC rewards for finding vulnerabilities in any part of the Ethereum software stack. This has undoubtedly contributed to the reliability and security of Ethereum and the confidence of the Ethereum community in the technology. The bounty program is currently still active and there is no end date planned. 
+
+* https://blog.ethereum.org/2015/03/20/juttas-update-bug-bounty-program-security-audit/.
+
+DEVgrants program
+--------------------------------------------------------------------------------
+In April 2015, `the DEVgrants program <https://blog.ethereum.org/2015/04/07/devgrants-help/>`_ was announced, which is a program that offers funding for contributions both to the Ethereum platform, and to projects based on Ethereum. Hundreds of developers were already contributing their time and thinking to Ethereum projects and in open source projects. This program served to reward and support those developers for their contributions. The DEVgrants program continues to operate today and funding of the program was recently renewed in January 2016.
+
+* https://blog.ethereum.org/2016/01/08/d%CE%BEvgrants-update-new-funding/
+
+Security Audit
+--------------------------------------------------------------------------------
+The Ethereum security audit began at the end of 2014 and continued through the first half of 2015. Ethereum engaged multiple third party software security firms to conduct an end-to-end audit of "everything" (from networking to the Ethereum VM to syncing layer to Proof-of-Work, etc). The audits uncovered security issues that were addressed and tested again, but ultimately resulted in a more secure platform.
+
+* https://blog.ethereum.org/2015/07/07/know-ethereum-secure/
+
+
```

Comment:
Olympic testnet phase and various bounties should be mentioned here
Frankly the fact that frontier was late should get a mention since high expectations and waiting was dominanating spring 2015